src/utils.ts

```diff
@@ -157,7 +157,19 @@ export function openDDLEditor(vdbName: string) {
 	}
 }
 
-export function createOrUpdateLocalReferenceFile(refName: string, refKey: string, entryValue: string, entryType: string) {
+export function createOrUpdateLocalReferenceFile(refName: string, variableName: string, variableValue: string, refType: string) {
 	// TODO: create or update the entry reference in a local yaml file for configMap OR secret format
 	// TODO: implement me!
 }
+
+export function doesLocalReferenceFileExist(vdbFile: string, refName: string, refType: string): boolean {
+	// TODO: check if a file exists for the given type and name in the same folder than the vdbFile
+	// TODO: implement me!
```

Comment:
the line `TODO implement me` is not very useful. The TODO implies that it would be better to be implemented already.

Reply:
Its a comment Aurelien. Only developers will see them and they will disappear once it got implemented.

Reply:
it remains better to not write in the first place